### ISSUE TITLE
Unable to save screenshots

### ISSUE THREAD

**Issue by CJampion** [ShareX/ShareX#3239]
I am getting this error when I attempt to save a screenshot.

![image](https://user-images.githubusercontent.com/31580027/37877767-b9e70cf6-302d-11e8-9115-999b7b3a17d1.png)

**Comment by CJampion** [comment]
Sorry, THIS is what I'm getting. Not sure what happened there. 
![image](https://user-images.githubusercontent.com/31580027/37877770-c69bbc80-302d-11e8-9eef-496b8a5451a1.png)

**Comment by Kiwi-1995** [comment]
Check the folder permissions.

**Comment by Jaex** [comment]
Most likely your anti virus using protected folder feature and blocked it.

**Comment by maximmax42** [comment]
If you use Avast, that's anti-ransomware screen blocking new version of ShareX, you can try either disabling it or adding the exe in the exceptions (but since there is no settings menu, no idea how, I only got it after like, 5-6 tries of save a file). If you use any other antivirus, should be similar.

**Comment by iKatt** [comment]
Disabling ransomware shield doesn't do anything. I still get the error.

**Comment by maximmax42** [comment]
Try to completely disable antivirus, then enable it again, then try to save a screenshot couple times, it should give you the "ransomware detected, what should I do?" window, where you can choose to allow it.

**Comment by stuntguy3000** [comment]
@CJampion @iKatt How you guys going with this issue?

**Comment by iKatt** [comment]
@maximmax32 @stuntguy3000 still the same issue. Downgrading to earlier versions doesn’t fix the issue either.

**Comment by stuntguy3000** [comment]
@iKatt A few questions

- What operating system do you use?
- What Antivirus do you have installed?

**Comment by iKatt** [comment]
@stuntguy3000 
Windows 10 Enterprise running the latest updates.
Avast Premier

**Comment by stuntguy3000** [comment]
To confirm, have you tried disabling Avast outright?

**Comment by iKatt** [comment]
Yeah I’ve tried and nothing happened. Ages ago I was able to upload a single mp4 but now I’m unable to upload and screen recordings. I can upload images just fine, however.

Sent from my iPhone

On 15 Apr 2018, at 4:05 pm, Luke Anderson <notifications@github.com<mailto:notifications@github.com>> wrote:


To confirm, have you tried disabling Avast outright?

**Comment by Eriond** [comment]
I had the same issue and resolved it by removing the "semi-marked" read-only attribute of the ShareX folder in MyDocuments: https://i.imgur.com/4vJWijE.png
Once you remove the square tick-in-the-box, verify by selecting "all subfolders", and click OK.
https://i.imgur.com/5vMgvPN.png
You may have to restart ShareX afterwards.
